qiskit/visualization/__init__.py

```diff
@@ -80,6 +80,21 @@
 
    pass_manager_drawer
 
+Pulse Visualizations
+====================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   pulse_drawer
+
+.. autosummary::
+   :hidden:
+   :toctree: ../stubs/
+
+   qiskit.visualization.pulse.qcstyle
+   qiskit.visualization.pulse.interpolation
```

Comment:
Sorry, I'm not sure why I suggested hidden qcstyle and interpolation module. It's better to take not use `:hidden:` as suggested in the [comment](https://github.com/Qiskit/qiskit-terra/pull/3902#discussion_r385222829) as it's not a known option for autosummary directive.

```suggestion
   pulse_drawer
   ~qiskit.visualization.pulse.qcstyle
   ~qiskit.visualization.pulse.interpolation
```